android/apollo/src/com/andrew/apollo/MusicPlaybackService.java

```diff
@@ -1410,15 +1418,18 @@ private void notifyChange(final String what) {
      * @param what The broadcast
      */
     private void updateRemoteControlClient(final String what) {
+
         int playState = mIsSupposedToBePlaying
-                ? RemoteControlClient.PLAYSTATE_PLAYING
+            ? RemoteControlClient.PLAYSTATE_PLAYING
+            : what.equals(PLAYSTATE_STOPPED)
+                ? RemoteControlClient.PLAYSTATE_STOPPED
                 : RemoteControlClient.PLAYSTATE_PAUSED;
```

Comment:
the block above should probably be rewritten with if statements for clarity.
Nested ternaries are a source of bugs and a pain to the eye.

I think, but I'm not sure, that we might be able to deduce that the player is stopped without having to add a new state.

If it's not playing, and it's not paused, isn't that the same as stopped?